packages/coinjoin/src/client/round/selectRound.ts

```diff
@@ -75,6 +79,24 @@ export const getAccountCandidates = (
 
         // TODO: double-check account max signed rounds, should be done by suite tho
 
+        const blameOfUtxos = arrayToDictionary(
+            account.utxos,
+            utxo => {
+                const blamedUtxo = blameOfInputs.find(i => i.id === utxo.outpoint);
+                return blamedUtxo?.roundId;
+            },
+            true,
+        );
+
+        if (Object.keys(blameOfUtxos).length > 0) {
+            log(`Found account candidate for blame round ~~${accountKey}~~`);
```

Comment:
It would be OK to treat blame rounds as normal rounds when it comes to `SessionPhases`, right?

Reply:
not my decision what should be displayed in UI, ask @hynek-jina

Reply:
Yes, user doesn't need to understand the concept of blame rounds..